build_tools/docker/context/setup_mmperf.sh

```diff
@@ -0,0 +1,52 @@
+#!/bin/bash
+
+# Copyright 2022 The IREE Authors
+#
+# Licensed under the Apache License v2.0 with LLVM Exceptions.
+# See https://llvm.org/LICENSE.txt for license information.
+# SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+#
+# Sets up `mmperf` (https://github.com/mmperf/mmperf).
+#
+# `mmperf` benchmarks matrix-multiply workloads on IREE and other backends such
+# as RUY, TVM, Halide, CuBLAS, etc. Some backends are included as submodules
+# in the `mmperf` repo and built from source, and other backends are expected
+# to already be installed.
+#
+# Usage:
+#    ./setup_mmperf.sh \
+#        <mmperf repo dir> \
+#        <mmperf sha>
+
+set -xeuo pipefail
+
+export REPO_DIR=$1
+export REPO_SHA=$2
+
+pushd ${REPO_DIR}
+git clone --jobs 8 --depth 1 --no-single-branch --recurse-submodules https://github.com/mmperf/mmperf.git
+pushd mmperf
```

Comment:
I actually needed --no-single-branch so that when I update the IREE submodule at each workflow run, I can just do a "git checkout origin/main". It's no longer relevant anyway since we're checking out a specific SHA now.